src/js/base/handalone.js

```diff
@@ -147,34 +150,45 @@ require(["pyret-base/js/runtime", "program"], function(runtimeLib, program) {
       runtime.setParam("current-checker", currentChecker);
     }
   };
-  postLoadHooks[main] = function(answer) {
-    var checkerLib = runtime.modules["builtin://checker"];
-    var checker = runtime.getField(runtime.getField(checkerLib, "provide-plus-types"), "values");
-    var getStack = function(err) {
-      console.error("The error is: ", err);
-      var locArray = err.val.pyretStack.map(runtime.makeSrcloc);
-      var locList = runtime.ffi.makeList(locArray);
-      return locList;
+  if (runMode) {
+    postLoadHooks[main] = function(answer) {
+      var definedValues = runtime.getField(answer, "defined-values");
+      var definedExitCode = definedValues["exit-code"];
```

Comment:
Ooh, I don't like this very much...  I *think* (first-draft speculation ahead) I'd rather a design where we had a runtime-provided function `set-exit-code : Number -> Nothing` that sets the eventual exit code of the program, or even `exit-now-with-code` which terminates the program now with the given exit code.  But having a specially-named identifier that could be provided by any module?  That seems too precious and unique to be a memorable way to exit the program...